src/language.rs

```diff
@@ -1,3 +1,27 @@
+// Portions of this code, highlighted below, are influenced by Difftastic (MIT licensed):
+//
+// Copyright (c) 2021-2022 Wilfred Hughes
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in all
+// copies or substantial portions of the Software.
```

Comment:
I don't think that this count as a “substantial portion of the software”. Feel free to cite Difftastic as an influence, but you probably don't need to include the licence here.

Reply:
Replaced with citations in a3276b8